### PR TITLE
feat(onboarding): trigger welcome bulle

### LINKED ISSUE #1047: afficher la bulle pour les nouveaux après l'onboarding
Actuellement, après installation la bulle du contributeur DisMoi s'affiche dès la 1ère phase de l'onboarding.

Il faudrait trouver un moyen pour qu'elle s'affiche sur la page d'abonnement listant les profils uniquement. 

En attendant la résolution, je mets la bulle en privé afin qu'elle ne gêne pas les prochains nouveaux utilisateurs.

Chrome prod : version 3.79.0

image

### PR DESCRIPTION
**Pull request by gregoirelacoste**
closes #1047
@MaartenLMEM dans le backoffice il suffira de paramétrer `pk_campaign=installed-contributors-list` pour afficher la welcome bulle

### CODE REVIEW

File: src/libs/domain/triggerWelcomeBulle.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { history } from '../../app/profiles/store';
+
+const triggerWelcomeBulle = (): void =>
```
Comment (lutangar) [inline src/libs/domain/triggerWelcomeBulle.ts:3]: Domain should not be dependent of the store... or dependence should at least be explicit :
```
const triggerWelcomeBulle = (history: History): void => history.push('?pk_campaign=installed-contributors-list');
```
Anyway, I don't think this -- navigation -- behavior is part of the domain.

Comment (gregoirelacoste) [reply src/libs/domain/triggerWelcomeBulle.ts:3]: ok

**Comment by gregoirelacoste** [comment]
@lutangar you can review this new way with withConnect

**Review by lutangar — APPROVED** [review]

**Pull request merged**